docs/pages/development/development-workflows.mdx

```diff
@@ -101,8 +100,102 @@ module.exports = () => {
+
+It's now available for all development clients `v0.9.0` and above. In order to install it, you'll need the most recent publish of `expo-updates`:
+
+<Terminal cmd={['$ npx expo install expo-dev-client expo-updates']} />
+
+#### Configure EAS Update
+
+If you have not yet configured EAS Updates in your project, you can find [additional instructions on how to do so here.](/eas-update/getting-started/)
+
+You can now view and load EAS Updates in your development build via the `Extensions` panel.
+
+## Set runtimeVersion in Expo config
+
+When you create a development build of your project, you'll get a stable environment to load any changes to your app that are defined in JavaScript or other asset-related changes. Other changes to your app, whether defined directly in **ios/** and **android/** directories or by packages or SDKs you choose to install, will require you to create a new build of your development build.
+
+To enforce an API contract between the JavaScript and native layers of your app, you should set the [`runtimeVersion`](/distribution/runtime-versions) value in the Expo config. Each build you make will have this value embedded and will only load bundles with the same `runtimeVersion`, in both development and production.
+
+## Build locally with Android Studio and Xcode
+
+> **warning** We do not recommend this method or modifying these **android/** and **ios/** directories directly, especially when creating a development build with EAS Build. For more information, see [Prebuild](/workflow/prebuild/).
+
+If you need to debug native code or you prefer to manage your native projects, you can build and distribute your project using the `npx expo run` commands. You'll need to set up or have access to Android Studio, Xcode, and related dependencies on your local computer.
+
+The [`npx expo run` commands](/workflow/expo-cli/#compiling) will create a new build, install it on your emulator/simulator or device, and start running it.
+
+<Tabs tabs={["For Android", "For iOS (macOS Only)"]}>
+
+<Tab>
+
+To build and run on an emulator:
+
+<Terminal cmd={['$ npx expo run:android']} />
+
+To build and run on a connected device:
+
+<Terminal cmd={['$ npx expo run:android -d']} />
+
+</Tab>
+<Tab>
+
+To build and run on a simulator:
+
+<Terminal cmd={['$ npx expo run:ios']} />
+
+To build and run on a connected device:
+
+<Terminal cmd={['$ npx expo run:ios -d']} />
+
+</Tab>
 
-You can set up your CI process to publish your project whenever a pull request is merged or updated and add a QR code that can be used to view the change in a compatible development build.
+</Tabs>
 
-[expo-preview-action](https://github.com/expo/expo-preview-action) can be used to implement this workflow in your project using GitHub Actions, or serve as a template in your CI of choice.
+Running the above commands generates native source code in their corresponding **android/** and **ios/** directories.
```

Comment:
I'm not remembering what it is (--clean maybe?), but the flag that will regenerate the native directories rather than taking them as a starting point for the generation would be good

Reply:
@tcdavis, isn't `--clean` used with `expo prebuild`? I'm not sure if there is any other flag to regenerate the native directories.

Reply:
Right on, I thought maybe `--no-build-cache`, but it looks like we don't actually provide an analogue for clean under the run commands.